Code/Engine/GameEngine/Animation/PropertyAnimComponent.h

```diff
@@ -9,7 +9,7 @@
 #include <GameEngine/GameEngineDLL.h>
 struct ezMsgSetPlaying;
 
-typedef ezComponentManagerSimple<class ezPropertyAnimComponent, ezComponentUpdateType::WhenSimulating> ezPropertyAnimComponentManager;
+using ezPropertyAnimComponentManager = class ezPropertyAnimComponent;
```

Comment:
clang-tidy messed up all component manager typedefs. I'll have to fix them manually.